src/main/java/org/generationcp/middleware/dao/oms/CVTermDao.java

```diff
@@ -1889,4 +1894,74 @@ protected String convertDataTypeToVariableDtoScale(final Integer dataTypeId) {
 		}
 	}
 
+	public List<Variable> searchAttributeVariables(final String query, final String programUUID) {
```

Comment:
@cuenyad 
Maybe you missed this comment yesterday.
If you are using constants for the scalars then, you should use same constants when building the query.

Reply:
done.